proto/p4/v1/p4runtime.proto

```diff
@@ -394,8 +394,19 @@ message Replica {
 // each replica's egress input metadata will be set to the respective values
 // programmed in the multicast group entry.
 message MulticastGroupEntry {
+  // Set multicast_group_id to MULTICAST_GROUP_ID_WILDCARD for wildcard reads
   uint32 multicast_group_id = 1;
   repeated Replica replicas = 2;
+
+  // We define a special reserved value for the multicast_group_id field. This
```

Comment:
This comment is only about `MULTICAST_GROUP_ID_WILDCARD`, maybe move it there.

Reply:
Done